src/main/java/mods/railcraft/common/carts/EntityCartChest.java

```diff
@@ -47,11 +50,6 @@ public boolean doInteract(EntityPlayer player, ItemStack stack, EnumHand hand) {
     }
 
     @Override
-    public EntityMinecart.Type getType() {
-        return EntityMinecart.Type.CHEST;
-    }
```

Comment:
This is intended to return `null`. The vanilla enum `EntityMinecart.Type` should be treated as deprecated under Forge, since its functionalities are covered by Forge-provided new methods. The reason not to throw an exception is because of the patch addressed [here](https://github.com/MinecraftForge/MinecraftForge/pull/3592/files#diff-7bc83f1d40c6fc8d3356e0ce7057cf9c).